.gitignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
I added it because we're not using it right now. If we need it later we can add it, but I'd rather not accidentally check in my package.json randomly.